indico/modules/events/client/js/components/AutoLinkerConfig.jsx

```diff
@@ -0,0 +1,135 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2023 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import autoLinkerConfigUrl from 'indico-url:events.autolinker_config';
+
+import PropTypes from 'prop-types';
+import React, {useState} from 'react';
+import {Button, Form, List, Message, Segment} from 'semantic-ui-react';
+
+import {handleSubmissionError} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+import {handleAxiosError, indicoAxios} from 'indico/utils/axios';
+
+import './AutoLinkerConfig.module.scss';
+import {rulePropTypes} from './propTypes';
+
+/**
+ * A simple rule-adding form
+ */
+function RuleEditor({regex, url, onChange}) {
+  function _onChange(key, value) {
+    const data = {regex, url};
+    data[key] = value;
+    onChange(data);
+  }
+
+  return (
+    <>
+      <Form.Input
```

Comment:
Maybe this field should use a monospace font?

PS: Does the monaco editor support a single-line / inline mode? If yes this could be a cool improvement for the future to get Python regex highlighting on the field... ;)

Reply:
Indeed.

As for the PS, I did consider it, but I didn't find any easy way to get a single-line Monaco instance.